libzecale/circuits/groth16_verifier/r1cs_gg_ppzksnark_verifier_gadget.tcc

```diff
@@ -15,10 +15,12 @@ r1cs_gg_ppzksnark_proof_variable<ppT>::r1cs_gg_ppzksnark_proof_variable(
     libsnark::protoboard<FieldT> &pb, const std::string &annotation_prefix)
     : libsnark::gadget<FieldT>(pb, annotation_prefix)
 {
+#if !defined(NDEBUG)
```

Comment:
What about switching to `#ifndef NDEBUG`? It's more compact.

Reply:
changed

Reply:
You could do the same here: https://github.com/clearmatics/zecale/blob/bls12-377-gadgets-miller/libzecale/circuits/groth16_verifier/r1cs_gg_ppzksnark_verifier_gadget.tcc#L109

Reply:
I don't have a strong preference here, I would only argue about consistency with the existing code (note that: we also use `#ifdef` instead of `#if defined()` in zeth, see: https://github.com/clearmatics/zeth/blob/master/prover_server/prover_server.cpp#L362 for eg).
We can still match/replace these to switch to the `#if defined()`/`#if !defined()` notation if you prefer, but I'd defer this for later though.

Reply:
Yeah, defering is no problem.  Actually, it's probably best to do this with the configuration changes in #19 where we remove the defaults from the library.  That will bring in zeth changes in which I've switched to this form (as it's a little clearer and avoids an extra macro to detect the error case).

Reply:
I see, perfect, let's do that!